string[] joint_name
float64[] command
